src/test/java/org/broadinstitute/hellbender/utils/io/IOUtilsUnitTest.java

```diff
@@ -289,4 +290,19 @@ private String getFirstLineAndDeleteTempFile(final File tempResourceFile) throws
         return resourceContentsFirstLine;
     }
 
+    @DataProvider
+    private Object[][] getIsHDF5TestFiles() {
+        return new Object[][] {
+                { getToolTestDataDir() + "/isValidHDF5.hdf5", true },
+                { getToolTestDataDir() + "/isTSV.tsv", false },
```

Comment:
You probably want test cases that have extension confusion as well since.  (Otherwise lets just check the extension instead of cracking open the file.)

Reply:
Done.